lib/node_modules/@stdlib/blas/ext/base/grev/benchmark/benchmark.js

```diff
@@ -38,13 +40,7 @@ var grev = require( './../lib/main.js' );
 * @returns {Function} benchmark function
 */
 function createBenchmark( len ) {
-	var x;
-	var i;
-
-	x = [];
-	for ( i = 0; i < len; i++ ) {
-		x.push( ( randu()*20.0 ) - 10.0 );
-	}
+	var x = gfillBy( len, new Float64Array( len ), 1, uniform( -100, 100 ) );
```

Comment:
Can we change this back to a generic array instead of `Float64Array`.

Reply:
Same comment.